### PR TITLE
Use correct syntax in lib/reacto/trackable.rb

### PR DESCRIPTION
**Pull request by madwort**
- looks like a typo?
- rubocop failed to parse this file using Ruby-2.0 parser

### CODE REVIEW

**Comment by meddle0x53** [comment]
Is this parsing with the Ruby-2.1 or 2.2 parsers? Maybe the mandatory keyword arguments are not valid with 2.0.
Was working with 2.0 while ago, and all was fine, but now it could be broken.

File: lib/reacto/trackable.rb [hunk 1]
```diff
@@ -620,7 +620,7 @@ def slice_before(pattern = NO_VALUE, &block)
       slice(pattern, type: :before, &block)
     end
 
-    def slice(pattern = NO_ACTION, type:, &block)
+    def slice(pattern = NO_ACTION, type: type, &block)
```
Comment (meddle0x53) [inline lib/reacto/trackable.rb:623]: Works with your fix, so will be merged.

**Comment by madwort** [comment]
with Ruby 2.0 because https://github.com/meddle0x53/reacto/blob/master/README.md#tested-with

**Comment by madwort** [comment]
Verified, the original version `type:,` parses correctly with `2.1`/`2.2`. Not a typo, a compatibility issue.

**Pull request merged**